README.md

```diff
@@ -19,7 +19,7 @@ On MacOS or Linux:
 MacOS/Linux:
 
     export OCI_HOME=<directory of Oracle instant client>
-    export OCI_LIB_DIR=$OCI_HOME
+    export OCI_LIB_DIR=$OCI_HOME/lib/
```

Comment:
Oh ok, should it be mentioned in the readme that this should be the directory containing `libocci.so` files?